integreat_cms/cms/utils/matomo_api_manager.py

```diff
@@ -352,10 +360,18 @@ def get_visits_per_language(self, start_date, end_date, period):
                         "data": list(offline_downloads.values()),
                     }
                 ]
+                # The dataset for online/web app downloads
+                + [
+                    {
+                        "label": _("Online/WebApp Downloads"),
```

Comment:
"Online download" sounds kind of strange to me, but not sure whether my suggestion of "online accesses" is really better :sweat_smile:

Reply:
Or simply "WebApp Downlods" ?

Reply:
Agreed ;-)

Reply:
I changed it to "Accesses" and "Zugriffe" everywhere.